packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/SelectedEntriesModal/index.js

```diff
@@ -42,25 +125,63 @@ const SelectedEntriesTableContent = () => {
           <Table.HeaderCell fieldSchemaType="string" label="name" name="name" />
         )}
       </Table.Head>
+      <Table.LoadingBody />
       <Tbody>
-        {rows.map((entry, index) => (
-          <Tr key={entry.id}>
-            <Body.CheckboxDataCell rowId={entry.id} index={index} />
-            <Td>
-              <Typography>{entry.id}</Typography>
-            </Td>
-            {shouldDisplayMainField && (
+        {!isLoading &&
```

Comment:
Hmm something seems off. The Table on the ListView also has an issue with the LoadingBody displaying at the same time as the content, for example when you unpublish. I just checked on main and we don't have that behavior there